mali: Support building against 5.3

Linux 5.3 changed the name of ktime_get_boot_ns to ktime_get_boottime_ns.

Nothing changed in that function otherwise though, so we just need to change
the function name.

Signed-off-by: Maxime Ripard <maxime.ripard@bootlin.com>

Upstream-Status: Backport
https://github.com/mripard/sunxi-mali/blob/master/patches/0021-mali-support-building-against-5.3.patch

Signed-off-by: Armin Kuster <akuster808@gmail.com>

Index: git/driver/src/devicedrv/mali/linux/mali_osk_time.c
===================================================================
--- git.orig/driver/src/devicedrv/mali/linux/mali_osk_time.c
+++ git/driver/src/devicedrv/mali/linux/mali_osk_time.c
@@ -53,7 +53,9 @@ u64 _mali_osk_time_get_ns(void)
 
 u64 _mali_osk_boot_time_get_ns(void)
 {
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 20, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 3, 0)
+	return ktime_get_boottime_ns();
+#elif LINUX_VERSION_CODE >= KERNEL_VERSION(4, 20, 0)
 	return ktime_get_boot_ns();
 #else
 	struct timespec tsval;
